### PR TITLE
✨ Add ProfileMenu

### PR DESCRIPTION
**Pull request by Dev-dfm**
Add elements for profile menu

### CODE REVIEW

**Review by lajanzen — APPROVED** [review]
lgtm 🚀

**Review by Maurizio-F — COMMENTED** [review]
Looks good! 👍

**Comment by J-Bannach** [comment]
lgtm 💯

File: src/app/components/Icons/HelpIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+function HelpIcon(): JSX.Element {
+  return (
+    <svg
+      id="icon-2"
```
Comment (phlgr) [inline src/app/components/Icons/HelpIcon.tsx]: `id` and `group` can usually be removed

File: src/app/components/Icons/LogoutIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="20.559"
+      height="20.56"
+      viewBox="0 0 20.559 20.56"
+    >
+      <g transform="translate(-0.014)">
+        <g transform="translate(0.014)">
+          <g id="Gruppe_20" data-name="Gruppe 20">
```
Comment (phlgr) [inline src/app/components/Icons/LogoutIcon.tsx]: This tag is not needed

**Review by daebert — COMMENTED** [review]
Good work!

**Review by phlgr — CHANGES_REQUESTED** [review]

File: src/app/components/ProfileMenuBar/ProfileMenuBar.module.css [hunk 3]
```diff
@@ -0,0 +1,12 @@
+.link {
+  text-decoration: none;
+  color: var(--text-color);
+  fill: var(--text-color);
+  display: flex;
+  align-items: center;
+  gap: 1.4em;
+  font-size: 1.1em;
+  border-bottom: solid 0.1em rgba(255, 255, 255, 0.26);
```
Comment (roabram) [inline src/app/components/ProfileMenuBar/ProfileMenuBar.module.css:9]: You could use a global variable instead, maybe?

**Review by roabram — COMMENTED** [review]
Nice job!

**Review by SGierszewski — APPROVED** [review]
LGTM! 💯

**Review by tmh9x — APPROVED** [review]

File: src/app/components/ProfileMenuBar/ProfileMenuBar.tsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import classes from './ProfileMenuBar.module.css';
+
+type LinkProps = {
+  icon: React.SVGProps<SVGSVGElement>;
+  text: string;
+};
+
+function ProfileMenuBar({ icon, text }: LinkProps): JSX.Element {
```
Comment (phlgr) [inline src/app/components/ProfileMenuBar/ProfileMenuBar.tsx]: I would expect a link prop to set the href of the component

**Review by phlgr — CHANGES_REQUESTED** [review]

**Review by lmachens — COMMENTED** [review]
Good job, I like to see a more describable pull request title and description.

**Review by phlgr — APPROVED** [review]
👍

**Pull request merged**